nextcloud-startup-script.sh

```diff
@@ -394,14 +394,13 @@ case *"$choice" in
         clear
         run_static_script automatic_updates
     ;;&
-    *"CookieLifetime"*)
-        clear
-        run_static_script cookielifetime
-    ;;&
     *)
     ;;
 esac
 
+# Nextcloud configuration
+bash $SCRIPTS/configuration.sh
```

Comment:
Maybe we could move that inside the startup script if it is not that important?
Could maybe be after lets encrypt and directly before additional_apps?

Reply:
I moved it to where I think it fits best.